ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/SubmissionValidator.scala

```diff
@@ -312,16 +312,16 @@ object SubmissionValidator {
   type StateMap = Map[DamlStateKey, DamlStateValue]
   type LogEntryAndState = (DamlLogEntry, StateMap)
 
-  private lazy val engine = Engine()
```

Comment:
@miklos-da, @mziolekda: Instead of allocating the `Engine` object here, it is passed in so that it can be shared between ledger api server and kv committer.

Reply:
Sounds good.